docs/manual/docs/cmd/spo/contenttype/contenttype-remove.md

```diff
@@ -0,0 +1,51 @@
+# spo contenttype remove
+
+Removes a content type from a site if not in use
+
+## Usage
+
+```sh
+spo contenttype remove [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --webUrl <webUrl>`|Absolute URL of the site where the content type is located
+`-i, --contentTypeId <contentTypeId>`|The ID of the content type to remove
+`-n, --name [contentTypeName]`|Content type name to remove if the content type id is not known. Either id or name must be specified but not both paramters
```

Comment:
The name of the option should match the long name, eg. `[name]` instead of `[contentTypeName]`